kong.yaml

```diff
@@ -1,47 +0,0 @@
-version: '3'
```

Comment:
I'd keep kong and infra isolated - I don't want to have to spin a massive stack just to get rabbitmq going locally. We should allow for the option here to run these separately.

Reply:
If you want to run just rabbitmq locally with this setup you can run `docker-compose up -d rabbitmq`. You don't have to start everything at once, this just gives you the option to do so.